### PR TITLE
TST Add some code coverage

### PR DESCRIPTION
**Pull request by VincentRouvreau**

### CODE REVIEW

File: icubam/messaging/sms_sender_test.py [hunk 1]
```diff
@@ -21,6 +21,13 @@ def test_from_string(self):
       sms_sender.get(self.config, 'Fake'), sms_sender.FakeSender)
     self.assertIsInstance(
       sms_sender.get(self.config, 'fake'), sms_sender.FakeSender)
+    with self.assertRaises(ValueError):
```
Comment (rth) [inline icubam/messaging/sms_sender_test.py:24]: might be good to use `assertRaisesRegex` with part of the error message, to be more specific.

File: icubam/messaging/sms_sender_test.py [hunk 2]
```diff
@@ -21,6 +21,13 @@ def test_from_string(self):
...
+      sms_sender.get(self.config, 'dummy_sms_sender_name')
+
 
   def test_from_config(self):
     self.assertIsInstance(sms_sender.get(self.config), sms_sender.FakeSender)
+
+  def test_send_from_fake_sender(self):
+    ms = sms_sender.get(self.config, 'Fake')
+    ms.send("Fake phone number", "Hello from tests")
```
Comment (rth) [inline icubam/messaging/sms_sender_test.py:33]: Maybe just add an inline comment explaining what this does: i.e. I hope this actually won't send a SMS if the token is provided..

Comment (dulacarnold) [reply icubam/messaging/sms_sender_test.py:33]: Fake sender is safe.

Comment (VincentRouvreau) [reply icubam/messaging/sms_sender_test.py:35]: I add a comment to make it clearer

File: icubam/messaging/sms_sender_test.py (hunk 1 above)
Comment (VincentRouvreau) [reply icubam/messaging/sms_sender_test.py]: Good idea !

**Pull request merged**